extensions/blocks/opentable/edit.js

```diff
@@ -205,6 +228,7 @@ function OpenTableEdit( {
 
 	return (
 		<div className={ editClasses }>
+			{ noticeUI }
```

Comment:
The block is styled in a bit of a complicated way, with an overlay setting its height.
If we render the notice here, the embed would overflow outside the overlay.
You can observe this in one of the PR gifs, where the embed moves behind the inserter button.

I've tried a few options, and had success with moving it outside the edit wrapper.
```jsx
<>
	{ noticeUI }
	<div className={ editClasses }>
		{ ! isEmpty( rid ) && inspectorControls }
		{ ! isEmpty( rid ) ? blockPreview() : blockPlaceholder }
	</div>
</>
```

Awkwardly enough, when the block is centrally aligned, the text of the notice is centered as well.
We might want to fix it in CSS. 🤔

Reply:
Yep, this is much gooder thanks ... fixed the centre alignment too.